src/core/lib/channel/channel_args.cc

```diff
@@ -384,9 +474,9 @@ const grpc_channel_args* RemoveGrpcInternalArgs(const grpc_channel_args* src) {
   return dst;
 }
 
-const grpc_channel_args* UniquifyChannelArgKeys(const grpc_channel_args* src) {
-  if (src == nullptr) return nullptr;
-  std::map<absl::string_view, const grpc_arg*> values;
+ChannelArgs UniquifyChannelArgKeys(const grpc_channel_args* src) {
+  if (src == nullptr) return ChannelArgs();
+  ChannelArgs output;
   std::map<absl::string_view, std::vector<std::string>> concatenated_values;
```

Comment:
This could presumably use `absl::string_view` instead of `std::string` in the vector, couldn't it?

Reply:
done